dhis-2/dhis-api/src/main/java/org/hisp/dhis/webapi/controller/event/mapper/OrderParamsHelper.java

```diff
@@ -54,4 +59,21 @@
                 .build() )
             .collect( Collectors.toList() );
     }
+
+    public static List<String> validateOrderParams( List<OrderParam> orderParams,
+        Map<String, TrackedEntityAttribute> attributes )
+    {
+        List<String> errors = new ArrayList<>();
+        if ( orderParams != null && !orderParams.isEmpty() )
```

Comment:
I think the case of `orderParams.isEmpty()` is covered by the foreach loop, so you can remove it.

As you might know, I like guard clauses to reduce nesting 😋 You could turn it into

```java
if (orderParams == null) {
return errors;
}

for ( OrderParam orderParam : orderParams ) {
...
```